cartography/intel/aws/ec2.py

```diff
@@ -111,6 +146,22 @@ def load_ec2_instances(session, data, region, current_aws_account_id, aws_update
     SET r.lastupdated = {aws_update_tag}
     """
 
+    ingest_key_pair = """
+    MERGE (keypair:KeyPair:EC2KeyPair{arn: {KeyPairARN}})
```

Comment:
Change this to `MERGE (keypair:KeyPair:EC2KeyPair{arn: {KeyPairARN}, id: {KeyPairARN}})` for the reason above

Reply:
Changed.